packages/web-app-files/src/services/folder/loaderSharedWithOthers.ts

```diff
@@ -30,6 +30,8 @@ export class FolderLoaderSharedWithOthers implements FolderLoader {
 
       resources = yield resources.json()
       resources = resources.ocs.data
+      // filter out shares via link only
+      resources = resources.filter((r) => r.share_type === 0)
```

Comment:
@diocas you can omit this commit, we have https://github.com/owncloud/web/blob/7e8e94964c2b51470459db0b9aa13515f8aad052/packages/web-app-files/src/services/folder/loaderSharedWithOthers.ts#L34 on master now which doesn't even include link shares in the request :)

Reply:
Perfect, updated!